src/server/services/generation/generation.types.ts

```diff
@@ -22,11 +22,13 @@ export namespace Generation {
     removedForSafety: boolean;
     jobToken?: string;
     duration?: number | null;
+    remixedFromImageId?: number;
   };
 
   export type Data = {
     params?: Partial<Params>;
     resources: Resource[];
+    remixedFromImageId?: number;
```

Comment:
Dunno if you discussed with koen, but I'd guess this could go into the params instead of being a thing of it's own